pom.xml

```diff
@@ -693,6 +695,7 @@
             <environmentVariables>
               <LIVY_TEST>true</LIVY_TEST>
               <SPARK_HOME>${spark.home}</SPARK_HOME>
+              <LIVY_LOG_DIR>${livy.log.dir}</LIVY_LOG_DIR>
```

Comment:
Will this change the log output dir for integration test, previously it is under `miniLivyMain`.